ext/src/http/client/curl/http_operation_curl.cc

```diff
@@ -1248,7 +1292,7 @@ CURLcode HttpOperation::SendAsync(Session *session, std::function<void(HttpOpera
   async_data_->callback = std::move(callback);
 
   session->GetHttpClient().ScheduleAddSession(session->GetSessionId());
-  return code;
+  return CURLE_OK;
```

Comment:
I think this might be cleaner than returning `code` given that it can only be CURLE_OK, as otherwise it would've been returned to the caller on L1277